### PR TITLE
Update credits.jsx

### PR DESCRIPTION
**Pull request by Narulacodes**
### Resolves:
https://github.com/LLK/scratch-www/issues/1769

### Changes:

adds SFE (Siegel Family Endowment) to the list of supporting organizations

### staging QA:
- [ ] Siegel Family Endowment is listed in the credits, and the link goes to http://www.siegelendowment.org/

### Test Coverage:

screenshot 2018-01-22 at 7 47 09 pm

### CODE REVIEW

File: src/views/credits/credits.jsx [hunk 1]
```diff
@@ -257,6 +257,7 @@ var Credits = React.createClass({
                 <p>
                     <a href="http://www.nsf.gov/">National Science Foundation</a>,
                     <a href="http://www.scratchfoundation.org/"> Scratch Foundation</a>,
+                    <a href="https://http://www.siegelendowment.org"> Siegel Family Endowment </a>
```
Comment (Kenny2github) [inline src/views/credits/credits.jsx]: Add a comma after this line, remove the space before the `</a>`, change `https://http://` to `https://`, and you should be good to go!

Comment (redmangospros) [reply src/views/credits/credits.jsx]: Thanks.

**Comment by amazinigmech2418** [comment]
It says there's invalid JSON. @LiFaytheGoblin had that problem too.

**Comment by redmangospros** [comment]
I Never Changed The a json file

**Comment by amazinigmech2418** [comment]
I know @redmangospros . It was changed before you forked it.

**Comment by redmangospros** [comment]
The last edit in credits was a pul request by @kenny2github

File: src/views/credits/credits.jsx [hunk 2]
```diff
@@ -257,6 +257,7 @@ var Credits = React.createClass({
                 <p>
                     <a href="http://www.nsf.gov/">National Science Foundation</a>,
                     <a href="http://www.scratchfoundation.org/"> Scratch Foundation</a>,
+                    <a href="https:/www.siegelendowment.org"> Siegel Family Endowment </a>
```
Comment (chrisgarrity) [inline src/views/credits/credits.jsx]: Please add a comma at the end of the line and remove the space before the closing `</a>`

**Comment by chrisgarrity** [comment]
@Narulacodes the updated Welsh translation fixed the build issue (I restarted the build)

**Comment by Narulacodes** [comment]
Thanks, I think we;re all ready to merge and review, |I had trouble with npm start and test

**Comment by Narulacodes** [comment]
Ok Thx.

**Comment by GreenBayRules** [comment]
I just wanted to point this out, if you look here in @Narulacodes supposed "test coverage" you can see that there is a comma with 2 spaces in front of it:
image
But, @chrisgarrity requested a change, saying this:
image

Just a thought

**Comment by Narulacodes** [comment]
Yeah, I forget to Change it, Had little trouble.
It wouldn't be that.

**Comment by GreenBayRules** [comment]
@Narulacodes Read the updated version

**Comment by Narulacodes** [comment]
Again, It would not be like that. I'm sure.

**Comment by Narulacodes** [comment]
When Will The E56 be merged

**Comment by amazinigmech2418** [comment]
Travis said `/home/travis/build/LLK/scratch-www/src/views/credits/credits.jsx
  321:1  error  Parsing error: Unexpected token`... I'll check the code to see if I can help.

**Comment by Narulacodes** [comment]
@amazinigmech2418 
Hmm. Travis shouldn''t be failing.
@kenny2github
Know what's wrong?

**Comment by Narulacodes** [comment]
@chrisgarrity

**Comment by rschamp** [comment]
@Narulacodes After merging the ES6 upgrade (#1790), there will be conflicts in our outstanding PRs. Because it will be complicated to merge, we will fix the conflicts ourselves, so don't worry about trying to fix it unless you want to!

**Comment by Narulacodes** [comment]
Okay, @rschamp. Thank you, I fixed the comflicts, when travis has errors.
Know how to fix?

**Comment by ktrvs** [comment]
@Narulacodes If you tested it locally, would the URL not be http://localhost:8333 instead of https://scratch.mit.edu/info/credits?

File: src/views/credits/credits.jsx [hunk 3]
```diff
@@ -7,6 +7,318 @@ const Page = require('../../components/page/www/page.jsx');
...
+                    <a href="http://saltstack.com/"> SaltStack</a>,
+                    <a href="https://github.com/etsy/statsd/"> StatsD</a>,
+                    <a href="http://www.ubuntu.com/"> Ubuntu</a>,
+                    <a href="https://www.varnish-cache.org/"> Varnish</a>.
+                </p>
+            </div>
+        );
+    }
+});
+=======
```
Comment (rschamp) [inline src/views/credits/credits.jsx]: Looks like this was left in from resolving the merge conflict. If we remove this, it should be good to go.

**Comment by Narulacodes** [comment]
Thank You! This is my Second PR!

**Pull request merged**